### PR TITLE
432 update docker simulation to choose vehicle type

### PR DESCRIPTION
**Pull request by 1Blademaster**

### CODE REVIEW

File: sitl_setup/run.sh [hunk 1]
```diff
@@ -1,9 +1,27 @@
 #!/bin/bash
+# Make sure line endings are LF for this file!!!
```
Comment (NexInfinite) [inline sitl_setup/run.sh:2]: Is this worth highlighting in readme?

Comment (1Blademaster) [reply sitl_setup/run.sh:2]: Don't think so tbh because `run.sh` should hardly ever need to be updated

Comment (NexInfinite) [reply sitl_setup/run.sh:2]: Aight sounds good

**Review by NexInfinite — APPROVED** [review]
LGTM

**Pull request merged**